README.md

```diff
@@ -1,2 +1,11 @@
-# relativepath.tech
-The RelativePath Website
+# React + TypeScript + Vite
+
+Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
+
+## Usage
+
+**Assumes you have Node installed. You can get it for your machine [here](https://nodejs.org/en/download)**
```

Comment:
Why is this an assumption? It would be better to write an assumption like "For our developer setup we have assumed Ubuntu 22.04 as our base OS and here is a script to get a developer setup and all the dependencies you need to run this site."

Reply:
This assumption is made because 90% of developers are using either Windows or Mac. Not everyone is willing to install Docker or other VM software to their machine. Installing Node is the simplest, low friction option. If developers have to jump through hoops just to be able to develop on this repo, I will be the only developer on this repo.